astro/src/styles/tokyo-night-dark.css

```
@@ -0,0 +1,114 @@
+}
+code.hljs {
+	padding: 3px 5px;
+}
+/*!
+  Theme: Tokyo-night-Dark
+  origin: https://github.com/enkia/tokyo-night-vscode-theme
+  Description: Original highlight.js style
+  Author: (c) Henri Vandersleyen <hvandersleyen@gmail.com>
+  License: see project LICENSE
+  Touched: 2022
+*/
```

Stylelint (17.2.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)